testcases/kernel/syscalls/read/read02.c

```diff
@@ -100,7 +108,7 @@ static void setup(void)
 
 #if !defined(UCLINUX)
 	outside_buf = SAFE_MMAP(0, 1, PROT_NONE,
-				MAP_PRIVATE | MAP_ANONYMOUS, 0, 0);
+				MAP_PRIVATE | MAP_ANONYMOUS, -1, 0);
```

Comment:
This change should not be required anymore.

Reply:
@davidchisnall  this test is actually failing without this patch. I tried in lsds/sgx-lkl.

Reply:
what is there error you get?

Reply:
@SeanTAllen  I shared the error above: https://github.com/lsds/ltp/pull/61#issuecomment-672563347

Reply:
And this error goes away with the change from 0 to -1?

Reply:
@hukoyu that is the error related to tmpfs and O_DIRECT that I detailed below.

Reply:
@hukoyu Sean is correct, the test is failing becaue of tmpfs and O_DIRECT issue